modules/openshift-developer-cli-using-command-completion.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * cli-reference/odo-cli-reference.adoc    
+
+[id="using-command-completion_{context}"]
+= Using command completion
+
+[NOTE]
+====
+* Currently command completion is only supported for bash, zsh, and fish shells.
+* Re-enable command completion if you either rename the {odo-title} executable or move it to a different directory.
+====
+
+{odo-title} provides a smart completion of command parameters based on user input. For this to work, {odo-title} needs to integrate with the executing shell.
+
+* To install command completion automatically
+.  Run:
++
+----
+odo --complete
+----
++
+.  Press `y` when prompted to install the completion hook.
+
+* To install the completion hook manually, add `complete -o nospace -C <full path to your odo binary> odo` to your shell configuration file.
+
+* To disable completion, run:
++
+----
+odo --uncomplete
+----
+
+After any modification to your shell configuration file, restart your shell.
```

Comment:
Does this part only apply to the second bullet? If so, maybe move that under there?